src/testRunner/unittests/convertCompilerOptionsFromJson.ts

```diff
@@ -539,5 +555,99 @@ namespace ts {
                 }
             );
         });
+
+        it("Convert tsconfig options when there are multiple invalid strings", () => {
+            assertCompilerOptionsWithJsonText(`{
+  "compilerOptions": {
+    "target": "<%- options.useTsWithBabel ? 'esnext' : 'es5' %>",
+    "module": "esnext",
+    <%_ if (options.classComponent) { _%>
+    "experimentalDecorators": true,
+    <%_ } _%>
+    "sourceMap": true,
+    "types": [
+      "webpack-env"<% if (hasMocha || hasJest) { %>,<% } %>
+      <%_ if (hasMocha) { _%>
+      "mocha",
+      "chai"
+      <%_ } else if (hasJest) { _%>
+      "jest"
+      <%_ } _%>
+    ]
+  }
+}
+`, "tsconfig.json",
+                {
+                    compilerOptions: {
+                        target: undefined,
+                        module: ModuleKind.ESNext,
+                        types: []
+                    },
+                    errors: [
+                        "tsconfig.json(5,5): error TS1136: Property assignment expected.\n",

```

Comment:
This test seems brittle given that it now depends on the exact positions of a huge number of parse errors. Maybe it should just be a test that we don't throw/crash on it?